src/main/java/com/diffplug/gradle/p2/AsMavenGroupImpl.java

```diff
@@ -68,7 +68,6 @@ public void run() throws Exception {
 			project.getLogger().lifecycle("p2AsMaven " + def.group + " is dirty.");
 		}
 		// else, we'll need to run our own little thing
-		FileMisc.cleanDir(dirP2());
 		FileMisc.cleanDir(dirP2Runnable());
```

Comment:
Should this be guarded on `append`?  Otherwise this looks great and ready to merge to me!